gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -1454,14 +1487,22 @@ private GoogleCloudStorageFileSystem createGcsFs(Configuration config) throws IO
     GoogleCloudStorageFileSystemOptions gcsFsOptions =
         GoogleHadoopFileSystemConfiguration.getGcsFsOptionsBuilder(config).build();
 
+    checkArgument(gcsFsOptions.getCloudStorageOptions().getAuthorizationMode()
```

Comment:
Does not look like this formatted with Google Java Format, may you reformat?

Reply:
yeah, seems like I missed running the script one last time.